nextflow.config

```diff
@@ -250,14 +250,13 @@ env {
 }
 
 // Set bash options
-process.shell = """\
-bash
-
-set -e # Exit if a tool returns a non-zero status/exit code
-set -u # Treat unset variables and parameters as an error
-set -o pipefail # Returns the status of the last command to exit with a non-zero status or zero if all successfully execute
-set -C # No clobber - prevent output redirection from overwriting files.
-"""
+process.shell = [
+    "bash",
+    "-C",         // No clobber - prevent output redirection from overwriting files.
```

Comment:
Not this change might not be working correctly - @mahesh-panchal reported this and proposed a different fix, see this thread and Mahesh's assoiciated alternative: https://github.com/nf-core/tools/pull/3416

Reply:
Thanks for the heads-up. I'll keep it in mind and wait for an official solution and merge it know